jdpostct.c

```diff
@@ -132,6 +132,11 @@ post_process_1pass (j_decompress_ptr cinfo,
   my_post_ptr post = (my_post_ptr) cinfo->post;
   JDIMENSION num_rows, max_rows;
 
+  /* read_and_discard_scanlines may call it with rows "available", but no buffer */
+  if (!output_buf) {
```

Comment:
NIT: libjpeg style is to use `(buf == NULL)` (probably because of some mythical system where this will fail due to NULL being implementation defined as a a type of zero value).